core/lib/dal/src/models/storage_transaction.rs

```diff
@@ -322,6 +322,85 @@ impl From<StorageTransaction> for Transaction {
     }
 }
 
+#[derive(Serialize, Deserialize, sqlx::FromRow)]
+pub struct StorageTransactionReceipt {
+    pub error: Option<String>,
+    pub tx_format: Option<i32>,
+    pub index_in_block: Option<i32>,
+    pub block_hash: Vec<u8>,
+    pub tx_hash: Vec<u8>,
+    pub block_number: i64,
+    pub l1_batch_tx_index: Option<i32>,
+    pub l1_batch_number: Option<i64>,
+    pub transfer_to: Option<sqlx::types::JsonValue>,
```

Comment:
Bikeshedding: IMO, using `serde_json::Value` instead of this alias would be more straightforward here; it's used in some other models.